examples/client/Locomotion/src/Components/DriverCard/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { RideInterface } from '../../context/newRideContext';
+import i18n from '../../I18n';
+import Stars, { StarIcon } from '../Stars';
+import {
+  RideViewTitleContainer,
+  RatingBarContainer,
+  RideViewSectionTitleText,
+  DriverRatingContainer,
+  DriverAvatarContainer,
+  DriverAvatar,
+  DriverDetailContainer,
+  DriverDetailText,
+  DriverSectionContainer,
+  DriverRatingText,
+  VehicleModelNameText,
+} from './styled';
+
+interface DriverCardProps {
+    activeRide: boolean;
+    ride: RideInterface;
+}
+
+const formatDriverRating = (rating: number): string | 0 => rating && rating.toFixed(1);
+const MAX_NAME_LENGTH = 20;
+
+const formatDriverName = (name: string): string => name && name.substring(0, MAX_NAME_LENGTH);
```

Comment:
wouldn't it be better do do an ellipses or something?

Reply:
Agree, but I just copied the code